tools/cicd/integration-tests.yaml

```diff
@@ -0,0 +1,38 @@
+# integration-tests.yaml
+steps:
+  # Check if the Docker image for Python 3.11 exists
```

Comment:
Given that the hatch images are created in the other build (https://github.com/Kaggle/kagglehub/blob/main/tools/cicd/cloudbuild.yaml#L3-L30), you can probably remove these 2 first steps here to keep it simple.